templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Create Manufacturer</h1>
+  <form action="" method="post" novalidate>
```

Comment:
The `action` attribute in the form tag is empty. It's recommended to explicitly specify the URL or use the `{% url %}` template tag for clarity and maintainability, even though it defaults to the current URL.